static/js/portico/signup.js

```diff
@@ -197,3 +197,29 @@ $(function () {
         this.parentNode.submit();
     });
 });
+
+$('#organization_type_select').on('change', function () {
```

Comment:
This should be scoped inside the previous block, like all the similar event handlers.